packages/aragon-api/src/index.js

```diff
@@ -148,6 +176,24 @@ export class AppProxy {
             pluck('result')
           )
         )
+      },
+      pastEvents: (options = {}) => {
+        const eventArgs = [
+          address,
+          jsonInterface.filter(
+            (item) => item.type === 'event'
+          ),
+          options
+        ]
+
+        return defer(
+          () => this.rpc.sendAndObserveResponse(
```

Comment:
Because we use `from()` in the `pastEvents` handlers, I think we should be using `sendAndObserveResponses()` for these past events instead.